app/assets/javascripts/editor.js

```diff
@@ -59,12 +56,37 @@ function (ko, models, tools, msg, validate, owned) {
             self
         );
 
+        self.editorTextShowCharsLeft = ko.computed(
+            function () {
+                if (self.editedAbstract() && self.editedAbstract().text()
+                            || $('#text').attr('maxLength') != undefined) {
```

Comment:
Although it (probably?) doesn't matter in this case: always use `!==` and `===` in. This saves the pain of thinking about crazy type conversion rules of `!=` and `==`.